ingest-app/test/cmr/ingest/test/services/jobs.clj

```diff
@@ -5,7 +5,39 @@
     [cmr.common.util :as u :refer [are3]]
     [cmr.ingest.services.jobs :as jobs]))
 
-(deftest create-query-params 
+(deftest create-query-params
   (is (= {"polygon" "-78,-18,-77,-22,-73,-16,-74,-13,-78,-18"
-          "concept-id" "G123-PROV1"} 
+          "concept-id" "G123-PROV1"}
          (#'jobs/create-query-params "polygon=-78,-18,-77,-22,-73,-16,-74,-13,-78,-18&concept-id=G123-PROV1"))))
+
+(deftest email-granule-url-list-test
```

Comment:
A bit excessive to have this test. But keep it since you already added it. 
FYI. you can call a private function with var quote syntax `#'`, e.g. `#'jobs/email-granule-url-list`

Reply:
actually adding forced me to clean something up, but sure, I'll do less testing next time.